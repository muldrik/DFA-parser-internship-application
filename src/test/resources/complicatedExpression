a = 10
b = 20
c = 30
d = (a + b) * (c - b) / (a-b-c)-(1)-((2))/2/2/2+a*b*c