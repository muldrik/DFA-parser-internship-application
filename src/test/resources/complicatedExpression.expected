d = (a + b) * (c - b) / (a - b - c) - (1) - ((2)) / 2 / 2 / 2 + a * b * c
